pa11ycrawler/html.py

```diff
@@ -38,117 +44,115 @@ def main():
     """
     parser = make_parser()
     args = parser.parse_args()
-    data_dir = os.path.expanduser(args.data_dir)
-    if not os.path.exists(data_dir):
+    data_dir = Path(args.data_dir).expand()
+    if not data_dir.isdir():
         msg = "Data directory {dir} does not exist".format(dir=args.data_dir)
         raise ValueError(msg)
-    data_filenames = [name for name in os.listdir(data_dir)
+    data_filenames = [name for name in data_dir.files()
                       if name.endswith(".json")]
     if not data_filenames:
         msg = "Data directory {dir} contains no JSON files".format(dir=args.data_dir)
         raise ValueError(msg)
-    output_dir = os.path.expanduser(args.output_dir)
-    if not os.path.exists(output_dir):
-        os.makedirs(output_dir)
+    output_dir = Path(args.output_dir).expand()
+    output_dir.makedirs_p()
 
     return render_html(data_dir, output_dir)
 
 
-COLOR_CLASSES = {
-    'error': 'danger',
-    'warning': 'warning',
-    'notice': 'info',
-}
-
-
-def get_code_info(code):
+def wcag_refs(code):
     """
-    Method for getting links to docs to present in reports.
-
-    Args: code (example: WCAG2AA.Principle3.Guideline3_2.3_2_1.G107)
-    Returns: a dict mapping technique to documentation url.
+    Given a `code` from pa11y, return a list of the WCAG references.
+    These references are always of the form: one or more capital letters,
```

Comment:
I wonder if you should just reference the other docstring rather than writing out the description again. [on the fence on that, but raising it here anyway.]